lib/tests/streamlit/report_context_test.py

```diff
@@ -63,6 +66,30 @@ def test_set_page_config_first(self):
         with self.assertRaises(StreamlitAPIException):
             ctx.enqueue(msg)
 
+    def test_disallow_set_page_config_twice(self):
+        """st.set_page_config must be called before other st commands
```

Comment:
nit: comment seems to have been copied from the test above and doesn't quite apply here